DemoApp-Java/app/src/main/AndroidManifest.xml

```diff
@@ -24,14 +20,29 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+
+        <activity
+            android:name=".ads.BannerAdActivity"
+            android:parentActivityName=".HomeActivity" />
         <activity
-            android:name="com.applovin.enterprise.apps.demoapp.ads.BannerAdActivity"
-            android:parentActivityName="com.applovin.enterprise.apps.demoapp.HomeActivity" />
+            android:name=".ads.LayoutEditorBannerAdActivity"
+            android:parentActivityName=".HomeActivity" />
         <activity
-            android:name="com.applovin.enterprise.apps.demoapp.ads.RewardedAdActivity"
-            android:parentActivityName="com.applovin.enterprise.apps.demoapp.HomeActivity" />
+            android:name=".ads.RewardedAdActivity"
+            android:parentActivityName=".HomeActivity" />
         <activity
-            android:name="com.applovin.enterprise.apps.demoapp.ads.InterstitialAdActivity"
-            android:parentActivityName="com.applovin.enterprise.apps.demoapp.HomeActivity" />
+            android:name=".ads.InterstitialAdActivity"
+            android:parentActivityName=".HomeActivity" />
+
+        <!-- AppLovin -->
+        <meta-data
```

Comment:
Any reason for moving them below the `activity` tags. I think they should be at the top, so that the devs can find it quicker.

Reply:
I think it was the autoformatter. I agree it's better on top.